contrib/babelfishpg_tsql/runtime/functions.c

```diff
@@ -3892,7 +3901,7 @@ bbf_pivot(PG_FUNCTION_ARGS)
 	MemoryContextSwitchTo(oldcontext);
 
 	oldcontext = MemoryContextSwitchTo(tsql_outmost_context);
```

Comment:
Why not directly switch to tsql_outmost_context in here ? Looks equal.

Reply:
removed that line. Resolved